docs/README.md

```diff
@@ -48,6 +48,19 @@ Repeat the above steps for the `system:node:$NODE_NAME` csr which will appear sh
 
 You can now do `oc get nodes`, and see the Windows node has joined the cluster.
 
+#### Unit testing
```

Comment:
If we are keeping the `test-unit` target, then you might want to add that if you are developing on windows you should run `make test-unit` to run the tests.

Reply:
Either that or we can remove the target